webapp/channels/src/types/external/html-to-react.d.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+/*
+Searched for `Parser`, `ProcessingInstructions`, and `ProcessNodeDefinitions` types on the html-to-react
+repository (https://github.com/aknuds1/html-to-react), but they assigned almost everything to an `any` type.
+*/
+
+declare module 'html-to-react' {
+    interface Node {
+        type: string;
+        name: string;
+        attribs: {
+            type: string;
+            checked: boolean;
+            'data-edited-post-id': string;
+            href: any;
+            mentionAttrib: string | undefined;
```

Comment:
Are `data-edited-post-id` and `mentionAttrib` needed? Could they be covered by the index signature?